### DISCUSSION TITLE
Getting started with Origin theme as a template

### DISCUSSION THREAD

**Discussion by farrulla-darze** [swellstores/community#330] [Help] [answered]
I'm having problems with the URL during this development phase. 
What should I place here on the `.env` file: `SWELL_STORE_URL=https://<your_store>.swell.store`?
I've tried placing a localhost:3000 but it doesn't work also tried placing it on the dashboard but it says it needs to be a valid URL. 
Here is the error I'm getting:
(base) ➜  origin-theme git:(master) yarn run dev
yarn run v1.22.19
$ nuxt

 WARN  Module @nuxtjs/stylelint-module is disabled due to incompatibility issues:   10:56:15 PM
 - [bridge] Nuxt bridge is required

ℹ Using Tailwind CSS from ~/assets/css/tailwind.css                   nuxt:tailwindcss 22:56:15
ℹ Merging Tailwind config from ~/tailwind.config.js                   nuxt:tailwindcss 22:56:15

 ERROR  Swell: unable to load settings (FetchError: request to http://localhost:3000/api/settings/all failed, reason: connect ECONNREFUSED 127.0.0.1:3000)

 FATAL  request to http://localhost:3000/api/settings failed, reason: connect ECONNREFUSED 127.0.0.1:3000

  at ClientRequest.<anonymous> (node_modules/node-fetch/lib/index.js:1505:11)
  at ClientRequest.emit (node:events:513:28)
  at Socket.socketErrorListener (node:_http_client:494:9)
  at Socket.emit (node:events:513:28)
  at emitErrorNT (node:internal/streams/destroy:157:8)
  at emitErrorCloseNT (node:internal/streams/destroy:122:3)
  at processTicksAndRejections (node:internal/process/task_queues:83:21)

   ╭───────────────────────────────────────────────────────────────────────────────╮
   │                                                                               │
   │   ✖ Nuxt Fatal Error                                                          │
   │                                                                               │
   │   FetchError: request to http://localhost:3000/api/settings failed, reason:   │
   │   connect ECONNREFUSED 127.0.0.1:3000                                         │
   │                                                                               │
   ╰───────────────────────────────────────────────────────────────────────────────╯

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by logeshswell** [comment, marked as answer]
> What should I place here on the .env file: SWELL_STORE_URL=https://<your_store>.swell.store?

If your store_id is `test` then
```env
SWELL_STORE_URL=https://test.swell.store
```
